cmd/shared.go

```diff
@@ -11,6 +16,14 @@ var (
 	undoFlag bool
 )
 
+func convertStringToBool(arg string) bool {
```

Comment:
changed to `stringToBool`. I don't see any uses of `2` instead of `to` anywhere in the golang builtins.